vendor/github.com/inngest/inngestgo/handler.go

```diff
@@ -341,6 +341,7 @@ func (h *handler) register(w http.ResponseWriter, r *http.Request) error {
 			f.EventBatch = map[string]any{
 				"maxSize": c.BatchEvents.MaxSize,
 				"timeout": c.BatchEvents.Timeout,
+				"key": c.BatchEvents.Key,
```

Comment:
The only reason I manually updated this was to make the tests pass (otherwise the test function doesn't include the key in its register request to the devserver).

We can remove this or update the entire vendor once we introduce SDK support for batch keys!

Reply:
You can make the change in `inngestgo` with a PR, and just vendor the hash for that commit.

Reply:
Great point, updated!